tests/scripts/components-configuration-crypto.sh

```diff
@@ -668,19 +668,13 @@ component_test_psa_crypto_config_accel_ffdh () {
     # start with full (USE_PSA and TLS 1.3)
     helper_libtestdriver1_adjust_config "full"
 
-    # Disable the module that's accelerated
-    scripts/config.py unset MBEDTLS_DHM_C
-
     # Build
     # -----
 
     helper_libtestdriver1_make_drivers "$loc_accel_list"
 
     helper_libtestdriver1_make_main "$loc_accel_list"
 
-    # Make sure this was not re-enabled by accident (additive config)
-    not grep mbedtls_dhm_ ${BUILTIN_SRC_PATH}/dhm.o
-
```

Comment:
It would be better to replace this by something like (not tested):
```
not grep mbedtls_psa_ffdh_key_agreement ${BUILTIN_SRC_PATH}/psa_crypto_ffdh.o
```